guides/common/modules/proc_using-container-registries.adoc

```diff
@@ -2,31 +2,73 @@
 = Using container registries
 
 Podman and Docker can be used to fetch content from container registries.
+Containers are pushed to {Project}'s container registry.
+The {Project} registry follows the Open Containers Initiative (OCI) specification, so you can push content to Katello by using the same methods that apply to other registries.
+For more information about OCI, see https://opencontainers.org/[Open Container Initiative Distribution Specification].
+
+.Prerequisites
+* To push content to an organization or product, a user must have permissions to create repositories.
+The push permission is `edit_products`.
+* To pull content, a user must have permissions as long as *unauthenticated pull* is set to off by default in the lifecycle environment.
+The pull permissions are `view_lifecycle_environments`, `view_products`, `view_content_views`.
 
 ifndef::orcharhino[]
 .Container registries on {SmartProxies}
-
 On {SmartProxies} with content, the https://github.com/Katello/smart_proxy_container_gateway[Container Gateway] {SmartProxy} plugin acts as the container registry.
 It caches authentication information from Katello and proxies incoming requests to Pulp.
 The Container Gateway is available by default on {SmartProxies} with content.
 endif::[]
 
-.Procedure
+.Limitations of pushing content to the {Project} container registry
+* You can only push content to the {ProjectServer} itself.
+If you need pushed content on {SmartProxies} as well, use {SmartProxy} syncing.
+* The pushed container name must contain only lowercase characters.
+* Unless pushed repositories are published in a content view version, they do not follow the Registry Name Pattern.
+This is to ensure that users can push and pull from the same path.
+* Users are required to push and pull from the same path.
+If a user uses the label-based schema, they pull using labels.
+If a user uses the ID-based schema, they pull using IDs.
 
-Logging in to the container registry:
+.Procedure
+. Logging in to the container registry:
++
 [options="nowrap", subs="+quotes,attributes"]
 ----
 # podman login {foreman-example-com}
 ----
 
-Listing container images:
+. Listing container images:
++
 [options="nowrap", subs="+quotes,attributes"]
 ----
 # podman search {foreman-example-com}/
 ----
 
-Pulling container images:
+. Pulling container images:
++
 [options="nowrap", subs="+quotes,attributes"]
 ----
 # podman pull {foreman-example-com}/my-image:<optional_tag>
 ----
+
+Pushing container images to the {Project} container registry:
+
+* To indicate which organization, product, and repository the container image belongs to, include the organization and product in the container repository name.
+* All labels are composed of entirely lowercase characters regardless of actual label capitalization.
```

Comment:
Is this about an action that users must take? Like "Compose all labels entirely of lowercase chars"?

I'm asking because if this an action item for users, it should be written in imperative voice. If it's just an FYI item, then perhaps it doesn't need to be included at all..? It's a bit confusing: I'm reading this and I don't know if I'm being asked to do something here.

Reply:
It isn't an action by the user so much as that is how the labels are composed. I will remove it.